kfdef/kfctl_aws.v1.0.0.yaml

```diff
@@ -28,6 +28,14 @@ spec:
         name: manifests
         path: istio/istio
     name: istio
+  - kustomizeConfig:
+      parameters:
```

Comment:
I think you want to modify the files in
https://github.com/kubeflow/manifests/tree/master/kfdef/source/master

So that when the script is to regenerate the files you pull in the changes.

Reply:
Ah. Forget these manifests are generate ones after the latest PR. I will make the change